docs/clientapi.md

```diff
@@ -124,7 +124,7 @@ order.
 Request that the IRC server sends a list of available channels. Extra parameters
 will be sent.
 
-##### `.channel(channel_name)`
+##### `.channel(channel_name, key)`
```

Comment:
it's optional, so wrap it in `[]` same as the other commands

Reply:
Thanks, need to review the doc specs for future reference.